modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -43,6 +43,7 @@
           :value="next"
           :disabled="field.readOnly"
           :has-relationship-schema="!!field.schema"
+          :image-editor="field.editor === imageRelationshipComponent"
```

Comment:
This should not be hardcoded, the purpose of field.editor is to make the component name configurable, hardcoding this check for a specific editor would defeat that.